### ISSUE TITLE
Initiate pull/push via http

### ISSUE THREAD

**Issue by c2u5hed** [wp-sync-db/wp-sync-db#63]
Is it possible to launch a pull/push by sending an http request (e.g. bitbucket webhook?)

**Comment by slang800** [comment]
yeah - I'm sure you could do that. I'd take a look at the code in [wp-sync-db-cli](https://github.com/wp-sync-db/wp-sync-db-cli) to see how to call the plugin, and then base your webhook implementation off of that.
